README.md

```diff
@@ -3,3 +3,59 @@
 `gitolize` is a simple tool that wraps arbitrary shell commands
 so that they get a working directory cloned from git and
 any changes made by them are pushed back to the git repository.
+
+## Usage
+
+```console
+house-reliability-engineering/gitolize$ ./gitolize.sh
+Usage: gitolize.sh [-a] [-b <branch>] [-c] [-l <local_directory>] [-m <message>] [-p <project>] [-r <repository>] [-s] [-v] [-w] [command...]
+house-reliability-engineering/gitolize$
+
+
+```
+
+### Options
+
+- `-a`:
+  when used with `-s`, ANSI escape sequences from the command's
+  output are preserved in the git commit message.
+  By default the ANSI escape sequences are stripped.
+
+- `-b <branch>`:
+  operate on a different branch than the
+  repository's default one.
+
+- `-c`:
+  print the git commit SHA on stderr.
+
+- `-l <local_directory>`:
+  use a particular directory for a git checkout instead
+  of a tmpdir. If this directory is a git checkout, then
+  the `-r` option is ignored.
+
+- `-m <message>`:
+  use a particular git message content instead of
+  the ran command.
```

Comment:
What about:
```suggestion
  the ran command output.
```

Reply:
Command output is added with `-s`.

Reply:
Does aec67687665d5f6100138ca460e207e1bba0c7fc make it clearer?

Reply:
Yes, thanks!